sound/soc/sof/ipc4-topology.c

```diff
@@ -476,6 +476,18 @@ static int sof_ipc4_widget_setup_comp_dai(struct snd_sof_widget *swidget)
 	ipc4_copier->data.gtw_cfg.node_id = SOF_IPC4_NODE_TYPE(node_type);
 
 	switch (ipc4_copier->dai_type) {
+	case SOF_DAI_INTEL_ALH:
```

Comment:
commit message: the description could use a few more words to explain the mapping and what this gateway does.

Reply:
I would recommend that we use 'SoundWire' with all letters instead of the 'sdw' acronym in the commit message.

Reply:
> I would recommend that we use 'SoundWire' with all letters instead of the 'sdw' acronym in the commit message.

done